### DISCUSSION TITLE
3.3.0

### DISCUSSION THREAD

**Discussion by LabhanshAgrawal** [vercel/hyper#6717] [Announcements]
V3.3.0

|   OS    | Installer |                                                                                                 |                                                                                                  |
| :-----: | :-------: | :---------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------: |
|   Mac   |    dmg    | [x64](https://github.com/vercel/hyper/releases/download/v3.3.0/Hyper-3.3.0-mac-x64.dmg) (Intel) | [arm64](https://github.com/vercel/hyper/releases/download/v3.3.0/Hyper-3.3.0-mac-arm64.dmg) (M1) |
|  Linux  |    deb    |      [x64](https://github.com/vercel/hyper/releases/download/v3.3.0/hyper_3.3.0_amd64.deb)      |     [arm64](https://github.com/vercel/hyper/releases/download/v3.3.0/hyper_3.3.0_arm64.deb)      |
|         |    rpm    |     [x64](https://github.com/vercel/hyper/releases/download/v3.3.0/hyper-3.3.0.x86_64.rpm)      |    [arm64](https://github.com/vercel/hyper/releases/download/v3.3.0/hyper-3.3.0.aarch64.rpm)     |
|         | AppImage  |      [x64](https://github.com/vercel/hyper/releases/download/v3.3.0/Hyper-3.3.0.AppImage)       |   [arm64](https://github.com/vercel/hyper/releases/download/v3.3.0/Hyper-3.3.0-arm64.AppImage)   |
|         |   snap    |     [x64](https://github.com/vercel/hyper/releases/download/v3.3.0/hyper_3.3.0_amd64.snap)      |                                                                                                  |
| Windows |    exe    |      [x64](https://github.com/vercel/hyper/releases/download/v3.3.0/Hyper-Setup-3.3.0.exe)      |                                                                                                  |

## Changelog
- Use v8 snapshots to cache dependencies (#6224)
- Passing `cursorAccentColor` down properly (#6538) @latin-1
- Dependency Updates

**Full Changelog**: https://github.com/vercel/hyper/compare/v3.2.3...v3.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/vercel/hyper/releases/tag/v3.3.0'>3.3.0</a>.</em>

**Comment by wzc0228** [comment]
hello, I downloaded latest version 3.3.0 in linux, however here is always display error.
![image](https://user-images.githubusercontent.com/100563747/187884175-3bf42924-e839-49d1-bda0-265cc676bc68.png)

**Reply by LabhanshAgrawal** [reply]
can you try with webGLRenderer set to false?

**Reply by wzc0228** [reply]
thanks
